waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/server/FederatedHMSHandler.java

```diff
@@ -1341,6 +1341,14 @@ public GrantRevokePrivilegeResponse grant_revoke_privileges(GrantRevokePrivilege
   public GrantRevokePrivilegeResponse refresh_privileges(HiveObjectRef hiveObjectRef, String authorizer,
                                                          GrantRevokePrivilegeRequest grantRevokePrivilegeRequest) throws MetaException, TException {
     DatabaseMapping databaseMapping = checkWritePermissions(hiveObjectRef.getDbName());
+    PrivilegeBag privilegeBag = grantRevokePrivilegeRequest.getPrivileges();
```

Comment:
could you update the Unit test as well please?

Reply:
Updated, pls review.